assets/src/edit-story/components/mediaPicker/useMediaPicker.js

```diff
@@ -153,5 +156,117 @@ export default function useMediaPicker({
     ]
   );
 
-  return openMediaPicker;
+  const openCropper = useCallback(
+    (evt) => {
+      trackEvent('open_media_crop_modal');
+      // If a user does not have the rights to upload to the media library, do not show the media picker.
+      if (!hasUploadMediaAction) {
+        if (onPermissionError) {
+          onPermissionError();
+        }
+        evt.preventDefault();
+        return;
+      }
+
+      const control = {
+        id: 'control-id',
+        params,
+        mustBeCropped,
+      };
+
+      // Create the media frame.
+      const fileFrame = global.wp.media({
+        button: {
+          text: buttonInsertText,
+          close: false,
+        },
+        states: [
+          new wp.media.controller.Library({
+            title,
+            library: wp.media.query({ type }),
+            button: {
+              text: buttonInsertText,
+              close: false,
+            },
+            multiple,
+            suggestedWidth: params.width,
+            suggestedHeight: params.height,
+          }),
+          new wp.media.controller.CustomizeImageCropper({
+            imgSelectOptions: calculateImageSelectOptions,
+            control,
+          }),
```

Comment:
Because of this change, Karma tests are currently failing with `Cannot read property 'Library' of undefined` errors. The video poster karma tests needs to be updated accordingly here:

https://github.com/google/web-stories-wp/blob/6687928bfc1eee50a07049ed73dcf835b960e56b/assets/src/edit-story/components/panels/design/videoAccessibility/karma/videoPoster.karma.js#L50-L69

Need to make sure `wp.media.controller`, `wp.media.controller.Library`, `wp.media.controller.CustomizeImageCropper`, and `wp.media.query` are properly defined.